### ISSUE TITLE
5.0.8 Patch Release

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#10108]
### Prerequisites

* [x] DEV: Create branch `release-x.x.x-rc.x` - tested in stable-5.0
  * [x] DEV: bump ocis version in necessary files
    * [x] DEV: `sonar-project.properties`
    * [x] DEV: `ocis-pkg/version/version.go`
  * [x] DEV: prepare changelog folder in 
  * [x] DEV: update production docker tag in the production.yml file in the continuous deployments
* [x] DEV: Check successful CI run on release branch
* [x] DEV: Create signed tag 
* [x] DEV: Check successful CI run on `v5.0.8` tag / BLOCKING for all further activity
* [x] DEV: Merge back release branch

### QA Phase

* [x] QA: Confirmatory testing (if needed)
- * [x] https://github.com/owncloud/enterprise/issues/6873
- * [x] https://github.com/owncloud/web/issues/11031
* [x] QA: Compatibility testing between 5.0.7 and 5.0.8
* [x] QA: Compatibility testing between 5.0.7 and 5.0.8 https://github.com/owncloud/ocis/issues/10108#issuecomment-2368669117
* [x] QA: Performance test  see result here https://grafana.k6.infra.owncloud.works/?orgId=1&refresh=15m&var-test_id=18b404f0-2cce-4da6-914a-7450b8729ee7
* [x] QA: Performance test  see result here https://grafana.k6.infra.owncloud.works/?orgId=1&refresh=15m&var-test_id=86fef651-a756-438e-af7d-d1c89638c439
* [x] QA: e2e with different deployment:
  * [x] QA: wopi against smoke test https://github.com/owncloud/web/pull/11621 
  * [x] run test again with new ocis https://drone.owncloud.com/owncloud/web/47789

**Comment by ScharfViktor** [comment]
note: we didn't port https://github.com/owncloud/ocis/commit/f29b2543333d8ba56e29474205a22de6527ae379 to `stable-5.0` so we get error: 
`{"level":"error","service":"notifications","error":"mail: no address","time":"2024-09-20T14:49:44+02:00","message":"parsing error, the 'smtp_sender' must be a valid single RFC 5322 address."}`

nothing critical - ocis still works

**Comment by ScharfViktor** [comment]
### Compatibility test between `owncloud/ocis:5.0.7` and `owncloud/ocis:5.0.8`

shares:
- [ ] public share (with an expiration date)
- [ ] share with user as role editor (with an expiration date)
- [ ] share with user as role viewer
- [ ] share with group "Users"

project space:
- [ ] uploaded folder & files
- [ ] trashed files
- [ ] share space with user
- [ ] public share space

files:
- [ ] uploaded folder & files
- [ ] version of files
- [ ] trashed files
- [ ] GDPR export

users
- [ ] non demo user with role admin
- [ ] non demo user with role user
- [ ] created new user
- [ ] disabled user
- [ ] created new group

settings
- [ ] language settings
- [ ] update logo админ
- [ ] new password

Remarks: no remarks

**Comment by butonic** [comment]
- [ ] merge https://github.com/owncloud/ocis/pull/10138 for clean reva version

**Comment by ScharfViktor** [comment]
### Compatibility test between `owncloud/ocis:5.0.7` and `owncloud/ocis:5.0.8` 
after reva bump
(https://hub.docker.com/layers/56940/ocis/dev/images/sha256-b09cb1925a386fef983dfda8f07322722491fe4a3a50f154270b240cb2004f3e?context=repo)

shares:
- [x] public share (with an expiration date)
- [x] share with user as role editor (with an expiration date)
- [x] share with user as role viewer
- [x] share with group "Users"

project space:
- [x] uploaded folder & files
- [x] trashed files
- [x] share space with user
- [x] public share space

files:
- [x] uploaded folder & files
- [x] version of files
- [x] trashed files
- [x] GDPR export

users
- [x] non demo user with role admin
- [x] non demo user with role user
- [x] created new user
- [x] disabled user
- [x] created new group

settings
- [x] language settings
- [ ] update logo админ
- [x] new password

Remarks: the logo didn't load  
<img width="328" alt="image" src="https://github.com/user-attachments/assets/306d806a-0bb7-470a-b9df-58ef0d56d8d5">

**Comment by ScharfViktor** [comment]
K6 tests: 
https://grafana.k6.infra.owncloud.works/?orgId=1&refresh=15m&var-test_id=86fef651-a756-438e-af7d-d1c89638c439
```
checks.........................: 100.00% ✓ 50924     ✗ 0    
     data_received..................: 24 GB   18 MB/s
     data_sent......................: 16 GB   12 MB/s
     http_req_blocked...............: avg=71.5µs   p(95)=7.42µs   p(99)=11.56µs  count=48498
     http_req_connecting............: avg=6.6µs    p(95)=0s       p(99)=0s       count=48498
     http_req_duration..............: avg=124.27ms p(95)=302.48ms p(99)=717.76ms count=48498
       { expected_response:true }...: avg=124.27ms p(95)=302.48ms p(99)=717.76ms count=48498
     http_req_failed................: 0.00%   ✓ 0         ✗ 48498
     http_req_receiving.............: avg=9.68ms   p(95)=595.08µs p(99)=426.48ms count=48498
     http_req_sending...............: avg=4.68ms   p(95)=42.27µs  p(99)=1.19ms   count=48498
     http_req_tls_handshaking.......: avg=58.9µs   p(95)=0s       p(99)=0s       count=48498
     http_req_waiting...............: avg=109.89ms p(95)=263.48ms p(99)=418.07ms count=48498
     http_reqs......................: 48498   36.323652/s
     iteration_duration.............: avg=11.73s   p(95)=19.57s   p(99)=23.94s   count=28467
     iterations.....................: 28467   21.320991/s
     vus............................: 1       min=0       max=300
     vus_max........................: 300     min=64      max=300
```

**Comment by ScharfViktor** [comment]
# 🎉  Infinite Scale 5.0.8

## Docker

`docker pull owncloud/ocis:latest` 

OR

`docker pull owncloud/ocis:5.0.8`

## Binaries

https://download.owncloud.com/ocis/ocis/stable/5.0.8/

**Comment by ScharfViktor** [comment]
https://drone.owncloud.com/owncloud/ocis/39468 was happy but I couldn't find 5.0.8 in the https://download.owncloud.com/ocis/ocis/stable

@micbar I remember it doesn't show up right away, but after a few tries I didn't find it. Do you know what I did wrong?

**Comment by micbar** [comment]
> https://drone.owncloud.com/owncloud/ocis/39468 was happy but I couldn't find 5.0.8 in the https://download.owncloud.com/ocis/ocis/stable
> 
> @micbar I remember it doesn't show up right away, but after a few tries I didn't find it. Do you know what I did wrong?

It will be published on the full hour

**Comment by micbar** [comment]
Binaries are published.

**Comment by micbar** [comment]
# Changelog

## Reva

  *   Fix [cs3org/reva#4761](https://github.com/cs3org/reva/pull/4761): Quotes in dav Content-Disposition header
  *   Fix [cs3org/reva#4853](https://github.com/cs3org/reva/pull/4853): Write upload session info atomically
  *   Enh [cs3org/reva#4701](https://github.com/cs3org/reva/pull/4701): Extend service account permissions
